fellow-submissions/devonte-duncan.md

```diff
@@ -0,0 +1,154 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+
+   To begin with, the declaration of all three of these variable types are automatically 
+   hoisted by javascript right before runtime, simply because they are variables.
+   As for their differences, variables declared with `const`, are designed to
+   be constant and therefore, JavaScript does not allow it's value to be reassigned.
+   On the other hand, using `let` to declare a variable does allow you to reassign 
+   its value later on. `let` and `const` are scoped variables, which means that its
+   existence can only be detected within its parent scope, whether that be global 
+   or local. As for `var` its properties are the same as `let`, except for the fact
```

Comment:
This is incorrect. Variables declared with `var` have a scope as well. Their scopes are defined by _functions_. We say they are _function scoped_.